version.py

```diff
@@ -75,6 +75,8 @@ def check_output(*popenargs, **kwargs):
 
 
 def get_version():
+    return "3.0.0"
```

Comment:
Lokks like something we need to cleanup.

Reply:
fixed